Userland/Libraries/LibGfx/Color.h

```diff
@@ -25,6 +25,12 @@ struct HSV {
     double value { 0 };
 };
 
+struct YUV {
```

Comment:
Do we actually need double precision for this HSL(A) below uses floats.

Reply:
I was sure HSL used floats 🤔

Reply:
It does not

https://github.com/SerenityOS/serenity/blob/27f5a18ce637598b83c5feec4e1392abe0100efc/Userland/Libraries/LibGfx/Color.h#L78-L79

Reply:
Ah, but the HSV data structure does. I guess I'll switch to floats completely, but that one is a confusing outlier.